docs/source/reference/numpysupported.rst

```diff
@@ -608,6 +608,7 @@ Math operations
  remainder           Yes          Yes
  mod                 Yes          Yes
  fmod                Yes          Yes
+ divmod (*)          ???          Yes
```

Comment:
Is `object mode` automatic?

Reply:
it should be!

Reply:
So I can, without any extra testing, set this to
```suggestion
 divmod (*)          Yes          Yes
```

Reply:
Yes, IIRC if it's working in `nopython` mode it'll work in `objmode`. Think the restrictions in this list are because of multiple return values not being supported until now (e.g. `frexp`, `modf`).